classes/Rijndael.php

```diff
@@ -48,24 +48,41 @@ public function encrypt($plaintext)
         if ($length >= 1048576) {
             return false;
         }
-        return base64_encode(mcrypt_encrypt(MCRYPT_RIJNDAEL_128, $this->_key, $plaintext, MCRYPT_MODE_CBC, $this->_iv)).sprintf('%06d', $length);
+        $ciphertext = null;
+        if (function_exists('mcrypt_encrypt') && version_compare(phpversion(), '7.1', '<')) {
+            $ciphertext = mcrypt_encrypt(MCRYPT_RIJNDAEL_128, $this->_key, $plaintext, MCRYPT_MODE_CBC, $this->_iv);
+        } elseif (function_exists('openssl_encrypt') && version_compare(phpversion(), '7.1', '>=')) {
```

Comment:
Why test for PHP 7.1 ? From the looks of it you should only check for PHP 5.3.3 in order to make sure that openssl_encrypt has the $iv parameter

> 5.3.3 | The iv parameter was added.